File: Lecture 3: If-Else, While loop & Lots of Patterns/Pattern32A.java
package com.company;

import java.util.Scanner;

public class Pattern32A {
    // Pattern 32 A
    public static void main(String [] args){
        /*
        Pattern 32 A
                                5
                                4 5
                                3 4 5
                                2 3 4 5
                                1 2 3 4 5
         */
        // Creating Scanner Object For taking inputs from the USER
        Scanner sc = new Scanner(System.in);
        // Take Number of ROWS and COLUMNS from the USER
        System.out.print(" Enter Number of ROWS for the Pattern: ");
        int n = sc.nextInt();
        pattern(n);
    }
    static void pattern(int n){
        // Procedure to print the Required Pattern
        for(int row =1; row <= n ; row++){

            int number = n - row + 1;
            for(int column = 1; column <= row ; column++){
                System.out.print( number + " ");
                number ++;
            }
            System.out.println();
        }
    }
}

File: 003_If-Else_While_loop_and_Lots_of_Patterns/Swift/patterns/Pattern32A.swift

import Foundation

// Function to print the specified pattern
func printPattern(_ n: Int) {
    for row in 1...n {
        var number = n - row + 1
        for _ in 1...row {
            print("\(number) ", terminator: "")
            number += 1
        }
        print("")
    }
}

// Taking user input for the number of rows
print("Enter Number of ROWS for the Pattern: ", terminator: "")
if let input = readLine(), let n = Int(input) {
    printPattern(n)
} else {
    print("Invalid input")
}
